gizmo.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import os.path
+
+
+def hello(name, country='Finland'):
+    print(f'Hello {name}, how are things in {country}?')
+
+
+def spell(word):
+    print('.'.join(list(word)))
```

Comment:
The point was to learn about for loops, but ok 🤷‍♂️

Reply:
I wanted to test what would happen if I didn't! :-)